kubernetes/README.md

```diff
@@ -4,70 +4,111 @@ This example explains how to deploy the 1Passwrd SCIM bridge on Kubernetes runni
+
+If you're not self-hosting your own redis server, you can set one up during this process. Otherwise, if you specified a custom redis server in the previous steps, you can skip this section.
+
+Use the `redis-deployment.yaml` and `redis-service.yaml` files with `kubectl` to deploy redis. If you have an existing redis instance, skip this step.
 
 Example:
+
 ```
 kubectl apply -f redis-deployment.yaml
 kubectl apply -f redis-service.yaml
 ```
 
-This will deploy a single redis instance listening on Kubernetes internal DNS `redis:6379`, which the SCIM Bridge will use for caching during operation. A redis instance is required when using the SCIM Bridge.
+This will deploy a single redis instance listening on Kubernetes internal DNS `redis:6379`, which the SCIM Bridge will use for caching during operation. 
 
-## Create your `scimsession` Kubernetes secret
 
-Next, we must create a Kubernetes secret containing the scimsession file. Using kubectl, we can read the scimsession file and create the secret in one command:
+## Deploy the SCIM bridge 
+
+Using the `op-scim-deployment.yaml` and `op-scim-service.yaml` files, deploy the 1Password SCIM bridge to your Kubernetes cluster.
+
+These files configure the 1Password SCIM Bridge to connect to the redis instance indicated by the args, and it deploys a load balancer to handle traffic on ports 80 and 443. Traffic on :80 is needed to perform the LetsEncrypt certificate challenges, after which all SCIM traffic will be served on :443.
+
 ```
-kubectl create secret generic scimsession --from-file=./scimsession
+kubectl apply -f op-scim-deployment.yaml
+kubectl apply -f op-scim-service.yaml
 ```
-Make sure to pass the filepath of the scimsession file that you downloaded.  The above command will look for the file in  this folder (the `/kubernetes/` folder) of the repository.
 
-## Deploy the SCIM bridge
 
-Using the `op-scim-deployment.yaml` and `op-scim-service.yaml` files, deploy the 1Password SCIM bridge to your Kubernetes cluster.
+## Finish configuring DNS
 
-NOTE: In order to obtain an SSL/TLS certificate for your SCIM Bridge instance, you must include a domain name in the `containers.args` field in the `op-scim-deployment.yaml` file. This will only succeed if a DNS record exists that points to your GCP load balancer. Please read the section about DNS records at the beginning of this example.
+As mentioned before, you'll need to configure you DNS after the fact, as configuring DNS requires the IP address of the deployed container.
 
-NOTE: If you are using an existing redis instance that is not running on `redis:6379`, add the `--redis-host=[host]` and `--redis-port=[port]` flags to `containers.args` in the deployment yaml file.
+It's important that you use the IP address of the container that runs the load balancer. The load balancer is what handles traffic for ports 80 and 443 for the SCIM bridge. Here are the mappings:
```

Comment:
What does this and the mappings have to do with configuring DNS? The mappings are generally only relevant in more technical setups.